docs/tutorials/work_with_authoring_versions.md

```diff
@@ -5,38 +5,51 @@ authoring content using the [compliance-trestle](https://ibm.github.io/complianc
 
 ## Step 1: Create a trestle workspace if you don't have one already
 
+This will initialize Trestle workspace that would contain all your templates and governed documents.
+
 ```bash
 mkdir my_workspace
 cd my_workspace
 trestle init
 >>> Initialized trestle project successfully in [user_path]/my_workspace
 ```
 
+The templates will be located in the `/my_workspace/.trestle/author/`. Please note that if you use MacOS or Linux distribution, by default `.trestle` will be hidden. You will need to reconfigure your system to show hidden folders.
```

Comment:
You will need to reconfigure your system to show hidden folders if you want to see them listed.